Microsoft.Toolkit.Uwp.UI.Controls/TokenizingTextBox/TokenizingTextBox.cs

```diff
@@ -134,12 +138,59 @@ private void AutoSuggestBox_SuggestionChosen(AutoSuggestBox sender, AutoSuggestB
 
         private void AutoSuggestBox_TextChanged(AutoSuggestBox sender, AutoSuggestBoxTextChangedEventArgs args)
         {
-            QueryTextChanged?.Invoke(sender, args);
```

Comment:
This is important so the implementer can actually know the text has changed and update the suggested items.